examples/loop_optimizations_service/example.py

```diff
@@ -0,0 +1,26 @@
+import compiler_gym
+import examples.loop_optimizations_service as loop_optimizations_service  # noqa Register environments.
+
+env = compiler_gym.make(
+    "loops-opt-py-v0",
+    benchmark="loops-opt-v0/add",
+    observation_space="ir",
+    reward_space="runtime",
+)
```

Comment:
Refactor this to use pattern:

```py
with compiler_gym.make(...) as env:
   ...
```

(i think I left the same comment on your other PR :) )